tests/jballerina-unit-test/src/test/resources/test-src/statements/block/block-stmt-semantics-negative.bal

```diff
@@ -13,3 +13,13 @@ function testRedeclareFunctionArgument (int value) returns (string) {
     }
     return "done";
 }
+
+function testRedeclareFunctionArgumentInBlockStmt (int value) returns (string) {
+    {
+        int value = 11;
+        testError tError = {message: "error", cause: error("errorMsg", code = "test")};
+        return "unreachable throw";
+        panic tError.cause;
+    }
+    return "done";
+}
```

Comment:
Can we also add negative tests for something like

```ballerina
function test() {
    {
        int i = 1;
        
        {
            int i = 2;
        }
    }
}
```

Reply:
And maybe uninitialized variable tests.

```ballerina
function test() {
    int i;
    {
        int j = i; // uninitialized i
        
        {
            i += 2; // // uninitialized i
        }
    }
}
```

Reply:
And for closures as well.

Reply:
Done